components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/inbound/IdentityMessageContext.java

```diff
@@ -37,6 +37,10 @@ public IdentityMessageContext(IdentityRequest request, Map<T1, T2> parameters) {
         this.request = request;
     }
 
+    public IdentityMessageContext(IdentityRequest request) {
+        this.request = request;
+    }
```

Comment:
Didn't understand why we need to call super() with parameters. This constructor doesn't take Map like its overloaded version. So we can only call the no-arg constructor of its super class.